demo/views/pages/document/document.js

```diff
@@ -44,32 +44,39 @@ class Document extends React.Component {
 
     if (content) {
       content = (
-        <div className="demo-document__content">
-          <Highlight innerHTML={ true }>
+        <div className='demo-document__content'>
+          <Highlight innerHTML>
             { marked(content) }
           </Highlight>
         </div>
       );
     } else {
-      spinner = <Row gutter="none"><Spinner columnAlign="center" size="small" className="demo-document__spinner"/></Row>;
+      spinner = (
+        <Row gutter='none'><Spinner
```

Comment:
```suggestion
  <Row gutter='none'>
    <Spinner
      columnAlign='center'
      size='small'
      className='demo-document__spinner'
    />
  </Row>
```

Reply:
Updated in https://github.com/Sage/carbon/pull/2496/commits/ac367dc8ea8d2c0b8524b08a78f38b0d13800f4e